orchestra/migrations/0060_auto_20160505_1349.py

```diff
@@ -6,6 +6,7 @@
 
 
 class Migration(migrations.Migration):
+    atomic = False
```

Comment:
only required for this migration and not others?

Reply:
Yeah, migrations check was failing with error message like `Cannot rename model atomically in SQLite`.

Reply:
I was not sure if we support `SQLite` or force any other RDBMS :)